src/ip/Address.h

```diff
@@ -220,7 +254,7 @@ class Address
      \param len byte length of buffer available for writing.
      \return pointer to buffer received.
      */
-    char* toUrl(char *buf, unsigned int len) const;
+    char *toUrl(char *buf, unsigned int len) const;
```

Comment:
Out of respect for reviewers, before asking for reviews, please look through PR diff and remove clearly unwanted out-of-scope changes like this one.